task7/index.js

```diff
@@ -3,24 +3,29 @@ const obj = [
     {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
 ]
 
-const div = document.getElementById("js-loading");
-const gif = document.createElement('img');
-div.appendChild(gif);
+const div = document.getElementById("js-parent");
+
+const loadingPlace = document.createElement("div");
+div.appendChild(loadingPlace);
```

Comment:
These two lines can also be included in the loading(). It is better to manage constants that are used only once by putting them in functions as much as possible, so that bugs are less likely to occur. This is because they cannot be read from the outside.

IMO
```js
const loading = () => {
  const loadingPlace = document.createElement("div");
  const gif = document.createElement("img");
  gif.src = "loading-circle.gif";
  div.appendChild(loadingPlace);
  loadingPlace.appendChild(gif);
};
```
However, this function is not yet complete.

I think it's inconvenient to just put this constant when removing.
This is because if you call the function from the outside, it will be undefined since you put it in the function. So what should we do?

Hint
You can use .then to get the loadingPlace when removing.

Reply:
Thank you for your comment.
I had a hard time not knowing my intention, but I learned a lot.

Reply:
It's perfect. Good work, sir.😆